lib/constants/collaboratorsInfoPDF.ts

```diff
@@ -0,0 +1,8 @@
+export const collaboratorsInfoPDF = {
```

Comment:
The conditional logic could be cleaned up by adding explicit conditional logic in collaboratorsInfoPDF for "songwriting" and "master," which allows for a more straightforward retrieval.

Reply:
I believe Faisal has already done this here, right?